chai-immutable.d.ts

```diff
@@ -0,0 +1,3 @@
+import * as chai from './typing/lib/chai-immutable';
+
+export = chai;
```

Comment:
On top of the missing EOF, would it make sense to call it `chaiImmutable` or `chaiImmutableTyping` instead of `chai`? (I am still totally inexperienced with TS so just trying to figure out what's in this var 😅)